xblockutils/base_test.py

```diff
@@ -1,4 +1,5 @@
 # -*- coding: utf-8 -*-
+# pylint: disable=missing-docstring
```

Comment:
But in all honesty - this is a disable I'd much rather see disabled on specific functions, OR, use the script I mentioned earlier to give yourself a bit of a buffer on pylint errors and leave these as future cleanup.